build.gradle

```diff
@@ -187,6 +187,10 @@ subprojects {
         }
     }
 
+    tasks.withType(Test) {
```

Comment:
i would remove this unless there are some benchmarks that show that this improves performance

Reply:
Ok, maybe I'll leave this value set to `2` as there is a room for test execution parallelism - I've been able to shave off about 10 seconds from the build total time. Looking at the test code - there are actually projects with more than one "long-running" test cases. The benefit from parallelising the tests slightly outperforms the punishment for forking the test runners (that means - the total running time is better but there is now obvious proof of it looking at the build scans).

Reply:
2 sounds like a good plan!

Reply:
oh, I was just changing something to re-trigger the build action. it's 2 now :)